src/utilities/forms.py

```diff
@@ -87,3 +90,20 @@ def clean(self, data, initial=None):
         #         pass
 
 # http://k
+
+
+class CustomFlatpageForm(FlatpageForm):
+
+    class Meta:
+        model = FlatPage
+        exclude = ('enable_comments', 'template_name',)
+
+        widgets = {
+            'content': SummernoteInplaceWidget(),
+            
+            # https://code.djangoproject.com/ticket/24453
+            'sites': forms.MultipleHiddenInput(),
+        }
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
Can remove overriding of `__init__()`?  It's not doing anything.

Reply:
removed